Cargo.lock

```diff
@@ -975,16 +976,27 @@ dependencies = [
 
 [[package]]
 name = "futures-core"
-version = "0.3.1"
+version = "0.3.3"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "79564c427afefab1dfb3298535b21eda083ef7935b4f0ecbfcb121f0aec10866"
+checksum = "b597b16aa1a19ce2dfde5128a7c656d75346b35601a640be2d9efd4e9c83609d"
 
 [[package]]
 name = "futures-core-preview"
```

Comment:
we don't have to remove these now, but I wonder where they're coming from.

Reply:
Huh that's strange. It doesn't come up with cargo-tree, and deleting/restoring Cargo.lock doesn't remove it either

Reply:
Ah okay, looks like a few more plugins are on the old stuff. Here's one:

https://github.com/nushell/nushell/blob/aa61689afd60e7b7a5d06830aeea9abf7bab8254/Cargo.lock#L2438

Reply:
Since this PR is close, I'm tempted to land it and then go upgrade the other plugins in a separate PR

Reply:
Up to you, it should be fine.